### PR TITLE
Update assoc.md

### PR DESCRIPTION
**Pull request by npocmaka**
Association for files with no extension

### CODE REVIEW

**Comment by PRMerger7** [comment]
@npocmaka : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/windows-commands/assoc.md [hunk 1]
```diff
@@ -33,8 +33,12 @@ assoc [<.ext>[=[<filetype>]]]
 
 ### Remarks
 
+- To make changes in associations you need administrator privileges.
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/assoc.md:36]: Cosmetic/grammar/codestyle:
```suggestion
- To make changes in associations, you need administrator privileges.
```

File: WindowsServerDocs/administration/windows-commands/assoc.md [hunk 2]
```diff
@@ -33,8 +33,12 @@ assoc [<.ext>[=[<filetype>]]]
...
+
 - To remove the file type association for a file name extension, add a white space after the equal sign by pressing the SPACEBAR.
 
+- To associatie files without extension to a file type use just dot (see the examples).
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/assoc.md:40]: Typo + comma + article:
```suggestion
- To associate files without extension to a file type, use just a dot (see the examples).
```

File: WindowsServerDocs/administration/windows-commands/assoc.md [hunk 3]
```diff
@@ -68,6 +72,18 @@ To send the output of **assoc** to the file assoc.txt, type:
 assoc>assoc.txt
 ```
 
+Associatie **.log** with no extension to text files:
```
Comment (illfated) [inline WindowsServerDocs/administration/windows-commands/assoc.md:75]: Copy-paste typo, maybe?
```suggestion
Associatie **.log** to text files:
```

**Comment by illfated** [comment]
Thank you for creating this PR. I would not have been as aware of how it could have been updated easily otherwise. Feel free to ask questions, if you have any.

**Comment by PRMerger15** [comment]
@npocmaka : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@illfated: I'm sorry - only the author of this article, @eross-msft, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment is based on an agreement with the team.

**Review by eross-msft — APPROVED** [review]

**Pull request merged**